### PR TITLE
Update ManufacturedPartsQInformtaion to 2.0.0

### LINKED ISSUE #435: [Model Update]: bamm:io.catenax.manufactured_parts_quality_information:1.0.0 to 2.0.0 with SAMM
## Update Reason
The use case quality group decided to upgrade this model to version 2.0.0:

    Use new versions of ESMF toolings
    Convert from BAMM to SAMM
    Make adaptions in model structure

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
--> new version will be 2.0.0
- [x] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by TorstenHirsch**
### Added
- metaInformation property added to root entity
- integration of the shared UUID characteristic and RegEx for Catena-X identifiers
- integration of BPNS as additional plant identifier using BPNS trait from shared aspect model
- introducing new characteristic for unique identifiers

### Changed
- changed all properties to optional except one unique identifier per entity

### Removed
- Manufacturing entity -> all properties are now located on ManufacturedPart entity

Closes #435 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 1]
```diff
@@ -0,0 +1,187 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:1]: for 2.0.0 version, new folder is missing. 
copyright headers are missing.
changes in release_notes required.

**Comment by TorstenHirsch** [comment]
@agg3fe Please check for MS-2 review
modeling_team

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-04T00:00:00"^^xsd:dateTime .
+
+:productionLine a samm:Property ;
+   samm:preferredName "Production line"@en ;
+   samm:description "On which production line was this part produced"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Line_1" .
+
+:addtionalInformation a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:159]: typo in property name `addtionalInformation`. i think better to fix it.

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+   samm:description "Defining the enumeration for the record status"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "new" "update" "delete" "same" ) .
+
+:UniqueID a samm:Characteristic ;
+   samm:preferredName "unique ID"@en ;
+   samm:description "This Characteristic describes a property that is truly unque either inside the Company that is providing the data or even in Catena-X dataspace. Only one property in a semantic entity should be a uniqueID. Unique ids can be used to link a data set with other semantic models."@en ;
+   samm:dataType xsd:string .
+
+:PositiveNumber a samm:Characteristic ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:193]: `preferredName` and `description` missing

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+   samm:dataType xsd:positiveInteger .
+
+:CountryCodeTrait a samm-c:Trait ;
+   samm:preferredName "Production Country Code Trait"@en ;
+   samm:description "Trait to ensure standard data format for country code"@en ;
+   samm-c:baseCharacteristic :CountryCodeCharacteristic ;
+   samm-c:constraint :CountryCodeRegularExpression .
+
+:ListOfAdditionalInformation a samm-c:List ;
+   samm:preferredName "Additional information list"@en ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:203]: `description` is missing

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+:RecordStatusEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Record status enumeration"@en ;
+   samm:description "Defining the enumeration for the record status"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "new" "update" "delete" "same" ) .
+
+:UniqueID a samm:Characteristic ;
+   samm:preferredName "unique ID"@en ;
+   samm:description "This Characteristic describes a property that is truly unque either inside the Company that is providing the data or even in Catena-X dataspace. Only one property in a semantic entity should be a uniqueID. Unique ids can be used to link a data set with other semantic models."@en ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:190]: typo `unque`

File: io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl [hunk 6]
```diff
@@ -0,0 +1,233 @@
+#######################################################################
+# Copyright (c) 2022, 2024 Robert Bosch GmbH
+# Copyright (c) 2022, 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2022, 2024 Volkswagen AG
+# Copyright (c) 2022, 2024 ZF Friedrichshafen AG
+# Copyright (c) 2022, 2024 SAP SE
+# Copyright (c) 2022, 2024 Siemens AG
+# Copyright (c) 2022, 2024 Contributors to the Eclipse Foundation
+# Copyright (c) 2022 Mercedes Benz AG
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.0.0/ManufacturedPartsQualityInformation.ttl:9]: missing 2024 entry?

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

**Comment by TorstenHirsch** [comment]
@agg3fe Please re-check
modeling_team

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**